ext/sftp/sink.go

```diff
@@ -135,3 +135,15 @@ func (s *SFTPSink) process() {
 		}
 	}
 }
+
+func (s *SFTPSink) writeFn(fh extcommon.FileHandler, b []byte) func() error {
```

Comment:
can we remove this?

Reply:
will use anonymous function instead. Just realize this haven't been used